config/defaults.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   'strictOrigin': true,
   'originsAllowed': ['https://apps.solid.invalid'],
   'dataBrowserPath': 'default',
-  'defaultContentType': 'text/turtle'
+  'defaultContentType': 'application/octet-stream'
```

Comment:
Is that setting used? Probably should be up to the resource mapper used.

Reply:
Yes, it's being forwarded to the resource mapper. (Because we removed all defaults there in the last PR)

Reply:
Ah yeah, my bad. I was only talking about the very specific defaults, such as the localhost and current folder one. octet-stream is the most basic MIME type, so that seems like a sensible default.